conformance/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/MappingRuleInterpreterOld.scala

```diff
@@ -0,0 +1,294 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.conformance.interpreter.rules
+
+import za.co.absa.enceladus.model.conformanceRule.MappingConformanceRule
+import org.apache.spark.sql.Dataset
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.SparkSession
+import za.co.absa.enceladus.dao.EnceladusDAO
+import za.co.absa.enceladus.conformance.CmdConfig
+import za.co.absa.enceladus.utils.transformations.ArrayTransformations
+import za.co.absa.enceladus.utils.transformations.ArrayTransformations.arrCol
+import org.apache.spark.sql.functions._
+import com.typesafe.config.ConfigFactory
+import za.co.absa.enceladus.conformance.datasource.DataSource
+import za.co.absa.enceladus.utils.schema.SchemaUtils
+import org.apache.spark.sql.api.java.UDF1
+import za.co.absa.enceladus.utils.error._
+import za.co.absa.enceladus.model.{Dataset => ConfDataset}
+
+import scala.util.Try
+import scala.util.control.NonFatal
+import org.apache.spark.sql.Column
+import za.co.absa.enceladus.conformance.interpreter.RuleValidators
+import za.co.absa.enceladus.utils.validation._
+
+case class MappingRuleInterpreterOld(rule: MappingConformanceRule, conformance: ConfDataset) extends RuleInterpreter {
```

Comment:
Since this old one is the default, does it make sense, to leave this one as `MappingRuleInterperter` and rename the new one to `MappingRuleInterpreterExperimental`?

Reply:
Yes, this will make much more sense

Reply:
Saša suggested that the experimental class will become the production one sooner or later. Then the class should be renamed again. So maybe the better approach is to rename the classes according to semantics of what they do, like `MappingRuleInterpreter` and `MappingRuleInterpreterNoExplode`. The "no explode" version is the new one because it does not explode, but expects the schema to be already exploded.

Reply:
That makes sense.